pyproject.toml

```
@@ -52,12 +52,14 @@ six = { version = ">=1.13.0", optional = true }
 httpx = { version = ">=0.23.3", optional = true }
 # Used by the Itential SSoT as a retry mechanism for HTTP failures in the AutomationGatewayClient.
 retry = "^0.9.2"
+# used for DNA Center integration
+dnacentersdk = "^2.5.6"
```

I see its grouped now, but shouldn't it be

```suggestion
dnacentersdk = { version = "^2.5.6", optional = true }
```

---

You are correct. I've fixed that.